community/bolt/src/docs/dev/serialization.asciidoc

```diff
@@ -420,6 +422,46 @@ Path (signature=0x50) {
 }
 ----
 
+The nodes (`N`) and relationships (`R`) are defined as follows:
+
+- `N` contains all the unique nodes in the path
+- `R` contains all the unique relationships in the path
+- For `N`, the index is an integer commencing with 0 and incrementing by 1
+- For `R`, the index is an integer commencing with 1 and incrementing by 1
+- The value component for both `N` and `R` is the data corresponding to the node or relationship (as defined above)
+- In `N`, the first element must _always_ be the first node in the path (thus having 0 as the key)
+- No other _explicit_ rules apply as to either (i) the ordering of the other nodes in `N`, or (ii) the ordering of any of the relationships in `R`. However, while not required, it is recommended that implementations aim to list entities (i.e. nodes and relationships) in the order in which they are first encountered while traversing the path. This may help with the efficiency of reading and writing
+
+In the context of a transmission, the path is represented as a sequence of integers; we define `S` as the transmitted sequence, and `S'` as the full sequence.
```

Comment:
Has the concept "transmission" been defined previously? I would suggest doing so, if possible, for clarity